### PR TITLE
Tm implementation nupic.vision

### PR DESCRIPTION
**Pull request by jaredweiss**
Working implementation of TM in the saccade experiment

### CODE REVIEW

**Comment by jaredweiss** [comment]
@scottpurdy Here's the implementation of the saccade experiment using tm

**Comment by jaredweiss** [comment]
note: will not run currently w/ fast tm (it's broken in `nupic.research`. set the default to `general` instead of `fast` to test)

File: nupic/vision/mnist/run_saccades_experiment.pyw [hunk 1]
```diff
@@ -502,19 +503,23 @@ class MainGUI(object):
...
-    self.networkName = "{networksDir}/{network}".format(
-        networksDir=_NETWORK_DIR_NAME,
-        network=os.listdir(_NETWORK_DIR_NAME)[-1])
-    self.testingNetwork.loadFromFile(self.networkName)
+    #self.networkName = "{networksDir}/{network}".format(
+    #    networksDir=_NETWORK_DIR_NAME,
+    #    network=os.listdir(_NETWORK_DIR_NAME)[-1])
+    #self.testingNetwork.loadFromFile(self.networkName)
+    self.testingNetwork = self.trainingNetwork
+    self.testingNetwork.networkSensor.setParameter("numSaccades", SACCADES_PER_IMAGE_TESTING)
```
Comment (scottpurdy) [inline nupic/vision/mnist/run_saccades_experiment.pyw]: line too long

**Comment by scottpurdy** [comment]
:+1:

**Comment by jaredweiss** [comment]
@scottpurdy 
added your comments

**Pull request merged**